packages/cli/src/constants.ts

```diff
@@ -16,6 +16,18 @@ export const RootFolderNode = new QTreeNode({
   default: "./",
 });
 
+export const EnvNode = new QTreeNode({
+  type: "text",
+  name: "env",
+  title: "Select or create an env for the project",
```

Comment:
better use full name "environment" in the user facing string.

Reply:
Fixed.